Lib/selectors.py

```diff
@@ -550,7 +597,38 @@ def unregister(self, fileobj):
                     pass
             return key
 
+        def modify(self, fileobj, events, data=None):
+            """Change a registered file object monitored events or attached data.
+
+            Parameters:
+            fileobj -- file object or file descriptor
+            events  -- events to monitor (bitwise mask of EVENT_READ|EVENT_WRITE)
+            data    -- attached data
+
+            Returns:
+            SelectorKey instance
+
+            Raises:
+            Anything that unregister() or register() raises
+            """
+            return super().modify(fileobj, events, data)
+
         def select(self, timeout=None):
+            """Perform the actual selection, until some monitored file objects are
```

Comment:
First docstring line should be a complete sentence < 72 chars long (a lot of tools expect that).

Reply:
Fix them all in that case please ;-)